tests/fast-integration/skr-test/test.js

```diff
@@ -1,63 +1,67 @@
-const {gatherOptions} = require('./');
-const {provisionSKR, deprovisionSKR} = require('../kyma-environment-broker');
-const {keb, gardener, director} = require('./helpers');
-const {initializeK8sClient} = require('../utils');
-const {unregisterKymaFromCompass, addScenarioInCompass, assignRuntimeToScenario} = require('../compass');
-const {oidcE2ETest, commerceMockTest} = require('./skr-test');
-const {KCPWrapper, KCPConfig} = require('../kcp/client');
+const {
+  gatherOptions,
+  withSuffix,
+  withInstanceID,
+  provisionSKRInstance,
+  director,
+  commerceMockTest,
+  oidcE2ETest,
+} = require('./index');
+const {getEnvOrThrow} = require('../utils');
+const {unregisterKymaFromCompass} = require('../compass');
+const {deprovisionSKRInstance} = require('./provision/deprovision-skr');
+const {
+  getSKRConfig,
+  prepareCompassResources,
+  initK8sConfig,
+} = require('./helpers');
 
-const kcp = new KCPWrapper(KCPConfig.fromEnv());
+const skipProvisioning = process.env.SKIP_PROVISIONING === 'true';
+const provisioningTimeout = 1000 * 60 * 30; // 30m
+const deprovisioningTimeout = 1000 * 60 * 95; // 95m
+let globalTimeout = 1000 * 60 * 70; // 70m
```

Comment:
Aren't 70 minutes a bit long for the tests under the consideration that provisioning and deprovisioning has its own timeout and is added on top to this global timeout ? 🤔

Reply:
Makes sense :)